### DISCUSSION TITLE
Next Auth with my own MongoDB data

### DISCUSSION THREAD

**Discussion by ChristophLeuker** [neuefische/questions#635] [Web]
Hey, the last few hours i tried to get a login for user (players in my case) over next-auth. I want to use the user data from my own MongoDB.
If anyone got time, can you please look at it? I dont know how this should work. Maybe its just a tiny problem or iam completely on the wrong way.
My MongoDB has a collection called players and a few data with always name, email und password. The connection itself to the project is working.

This is my auth branch : https://github.com/ChristophLeuker/Capstone-project-Throw-In-App/tree/nextauth.

**Comment by julialoeschel** [comment]
Hey Christoph, 
with that information I can not guess what I should look at. 
please let me know whats the situation and what are the files I should look at. 
- wht did you implement already? 
- what data do you need and where should it come from? 
- did you check the [nextAuth Documentation](https://next-auth.js.org/getting-started/client)?
